spec/classes/chrony_spec.rb

```diff
@@ -23,7 +23,7 @@
       end
 
       context 'chrony::config' do
-        case facts[:osfamily]
+        case facts['os']['family']
```

Comment:
I think facts are weird in puppet-rspec and it only converts the first level to symbols, not the nested ones:
```suggestion
        case facts[:os]['family']
```